modules/administration-guide/partials/proc_applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Prepare and apply the Bitbucket application link Secret as follows:
+
+.Prerequisites
+* The preceding procedure, Setting up the Bitbucket application link, has been completed.
```

Comment:
You don't know if users have looked at the preceding procedure. 

How about simply, `You have set up the Bitbucket application link.` 

You could link to the procedure if you'd like.

Reply:
@max-cx yes, I like these rewrites. Thank you for taking my feedback seriously :) 

FWIW, I hear you about links. They're not very encouraged but can make life much easier. I default to user experience, and I agree with you here: if users must complete these procedures in order, then link to the according procedures in prereqs. 

Good work :)

Reply:
Thank you for your support, Jake.